includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1489,4 +1577,86 @@ private function get_address_data_for_payment_request( $order ) {
 		];
 	}
 
+	/**
+	 * Collects the payment information needed for processing a payment intent.
+	 *
+	 * @param WC_Order $order The WC Order to be paid for.
+	 * @return array An array containing the payment information for processing a payment intent.
+	 */
+	private function prepare_payment_information_from_request( WC_Order $order ) {
+		// TODO: throw exception if any required information is missing.
```

Comment:
This one specifically was already solved. Removed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2753/commits/8098d581ea806a23d76e5afe43b160793f2a9f58

But, yup, it's probably confusing to have them around and we already have tasks for working on them. I'm removing the ones that aren't meant to be solved in this PR, which is all but one.

Reply:
Thanks @a-danae !

> It's probably confusing to have them around and we already have tasks for working on them

I personally think they're fine to have in this base PR to help indicate where we expect code from another issue to go. I just wanted to avoid shipping the feature with a bunch of TODO's that we added early on and then never got around to addressing 😄 

Happy to go with whatever you think is best!

Reply:
Makes complete sense 😄 

I've removed the TODO comments in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2753/commits/cf90c3c838baae55eab87de5098239b27f443fd1. We can decide where's the best place to handle the behaviors mentioned in those comments when working on the issues.